File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int i, j, k, h, t, m;
    
    do
    {
        h = get_int("enter the height: ");
    }
    while (h<1);//it worked untill here
    
    for(i=0;i<h;i++)
    {
        t=h-1-i;
        for(j=0; j<h;j++)
        {
            if(j<t)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
            printf("  ");
            m=h-t;
            for(k=0; k<m; k++)
            {
                printf("#");
            }
        printf("\n");
        //it works now
    }

}

File: pset6/mario/more/mario.py
from cs50 import get_int

h = 0
while not int(h) in range(1,9):
    h = get_int("Height:")

j = 0
for i in range (h):
    h -= 1
    j += 1
    print(" " * h + "#" * j + "  " + "#" * j)